core/test/configuration.ts

```diff
@@ -47,9 +47,6 @@ csharp:
     // output folder should be 'foo'
     assert.equal(cfg['output-folder'], 'foo');
 
-    // my-value should not get resolved here because the dependent variable is after
-    assert.equal(cfg['my-value'], '$(sample-value)');
```

Comment:
Seems unrelated, why does it need to be removed?

Reply:
It was breaking a test (configuration doesn't work like that anymore)... I dunno why I didn't catch it before.